identity-mapping.json

```diff
@@ -49,7 +49,7 @@
 			"value": "individualBiometrics"
 		},
 		"introducerBiometrics": {
-			"value": "introducerBiometrics"
+			"value": "parentOrGuardianBiometrics"
```

Comment:
why this needs to be changed, as per the 1.2.0.1 default id schema, what is the value these fields ?

Reply:
As per the below query
SELECT id, id_version, title, description, schema_json, status_code, add_props, effective_from, lang_code, is_active, cr_by, cr_dtimes, upd_by, upd_dtimes, is_deleted, del_dtimes
	FROM master.identity_schema order by cr_dtimes desc;
I checked the latest id schema in that parent details are present not introducer key word.